1_OCaml/TP/TP1.ipynb

```diff
@@ -28,6 +28,110 @@
     "Puis tester ces fonctions sur des arguments de votre choix."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    (* Pour résoudre cela via un algorythme, on créer une fonction et une variable pour compter le nombre d'itération d'un cycle*)
</pre><p>algorithme, pas algorythme</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mp2i-fsm/mp2i-2021/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>